scripts/adage_pancancer.py

```diff
@@ -55,6 +84,8 @@
                     help='The latent space dimensionality to test')
 parser.add_argument('-o', '--optimizer', default='adam',
                     help='optimizer to use', choices=['adam', 'adadelta'])
+parser.add_argument('-w', '--untied_weights', action='store_false',
```

Comment:
Just like above comment I think it would make more sense to rename this as `tied_weights` so the user knows to call this to tie the weights down.